tailwind.config.mjs

```diff
@@ -0,0 +1,19 @@
+import defaultTheme from 'tailwindcss/defaultTheme';
+export default {
+    content: ['./index.html', './src/**/*.{js,jsx,ts,tsx}'],
+    theme: {
+        extend: {
+            fontFamily: {
+                sans: ['"Raleway"', ...defaultTheme.fontFamily.sans],
+            },
+            colors: {
+                base: "#001236",
```

Comment:
`base` is reserved with classes such as `text-base` so it will be mixed up. I think we can rename this into `midnight`

Reply:
Good catch. Thanks🙏